spec/queue.js

```diff
@@ -0,0 +1,131 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import Queue from '../src/queue'
+
+chai.use(chaiChange)
+
+describe('Queue', () => {
+  it('exists', () => {
+    expect(Queue).to.be.a('function')
+  })
+
+  context('enqueue()', (element) => {
+    it('adds an element to the back of the queue.', () => {
+      const myQueue = new Queue()
+      myQueue.enqueue('foo')
+      myQueue.enqueue('poo')
+      myQueue.enqueue('boo')
+
+      expect(() => myQueue.enqueue('blue'))
+        .to.alter(() => myQueue.length(), { from: 3, to: 4 })
+    })
+
+    it('adds an element to the back of the queue.', () => {
```

Comment:
These it blocks are both parts of the same test - they should be combined into a single it block containing both expectations.